external/a2a/README.md

```diff
@@ -0,0 +1,27 @@
+# Teams: a2a
+
+<p>
+    <a href="https://www.npmjs.com/package/@microsoft/teams.a2a" target="_blank">
+        <img src="https://img.shields.io/npm/v/@microsoft/teams.a2a" />
+    </a>
+    <a href="https://www.npmjs.com/package/@microsoft/teams.a2a?activeTab=code" target="_blank">
+        <img src="https://img.shields.io/bundlephobia/min/@microsoft/teams.a2a" />
+    </a>
+    <a href="https://www.npmjs.com/package/@microsoft/teams.a2a?activeTab=dependencies" target="_blank">
+        <img src="https://img.shields.io/librariesio/release/npm/@microsoft/teams.a2a" />
+    </a>
+    <a href="https://www.npmjs.com/package/@microsoft/teams.a2a" target="_blank">
+        <img src="https://img.shields.io/npm/dw/@microsoft/teams.a2a" />
+    </a>
+    <a href="https://microsoft.github.io/teams.ts" target="_blank">
+        <img src="https://img.shields.io/badge/📖 docs-open-blue" />
+    </a>
+</p>
+
+A plugin enabling a2a on a Teams agent application
```

Comment:
might be nice to link to the official a2a repo/docs from here.

Reply:
Yeah, actually good idea to update this readme with how this thing works

Reply:
Updated readme